pkg/dockerutil/dockerutils_test.go

```diff
@@ -133,6 +133,13 @@ func TestComposeCmd(t *testing.T) {
 	assert.Error(err)
 }
 
+func TestCheckCompose(t *testing.T) {
```

Comment:
Our habit is to go ahead and put a description line (or more) in front of every function, not just non-test or exported functions.